osidb/models.py

```diff
@@ -2882,12 +2887,17 @@ class CVEv5Version(Version):
 
 
 class PackageVersions(PolymorphicModel):
+    """
+    This model is deprecated. Use Package instead. Delete when it's clear
+    that Package works correctly.
+    """
+
     # internal primary key
     uuid = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
 
     flaw = models.ForeignKey(
         Flaw,
-        related_name="package_versions",
```

Comment:
I am not sure about the future commits structure, but you should ensure that this change is done together (i.e in one commit) with creating the `Package` model and its `flaw = models.ForeignKey(Flaw, related_name="package_versions", on_delete=models.CASCADE,)`, so the `package_versions` does not disappear from the `Flaw` model.

Reply:
But it doesn't disappear, does it? It's moved to the new model. Ctrl+f for `related_name="package_versions"` in the commit.

Reply:
It can disappear in commitX and reappear in commitY, which should not happen, as we discussed earlier in our google space.

Reply:
I don't understand.

Please show commits where it disappears and commits where it reappears. I say that `related_name="package_versions"` is moved between models in commit 5cf938968f1737d20d2fdd2597155bd2b8dd2eb5

Reason for multiple commits is in the top-most commit in this PR.

Reply:
I did not say `it disappeared`, but `it can disappear` and `you should ensure`. I think this was actionable enough.